exporter/exporter.go

```diff
@@ -241,6 +247,21 @@ func (e *Exporter) collectImpl(ctx context.Context, out chan<- prom.Metric) erro
 			out <- e.newMetric("wan_rx_rate", prom.GaugeValue, wanIF.Statistics.Rxrate, deviceLabels...)
 			out <- e.newMetric("wan_tx_rate", prom.GaugeValue, wanIF.Statistics.Txrate, deviceLabels...)
 		}
+
+		// Ping metrics, if enabled
+		if e.extras.Ping {
```

Comment:
do you need to check this here? should be fine to use the nil-check in PingMetrics right? that way the `fetchDeviceData` is the only place it's handled, which i'd like.

Reply:
I'd rather leave it as is, because future extensions to the `models.DeviceStatistics` struct (hint: coming very soon) will require this conditional where it is now.